packages/dashboard/src/components/doors-app.tsx

```diff
@@ -1,32 +1,109 @@
-import { CardActions, Grid } from '@mui/material';
+import { Button, SxProps, Typography, useTheme } from '@mui/material';
 import { BuildingMap, DoorState } from 'api-client';
 import React from 'react';
-import {
-  DoorCard as BaseDoorCard,
-  DoorCardProps as BaseDoorCardProps,
-  DoorControls,
-} from 'react-components';
-import { DoorMode as RmfDoorMode } from 'rmf-models';
+import { doorModeToString, doorTypeToString } from 'react-components';
+import { DoorMode, DoorMode as RmfDoorMode } from 'rmf-models';
 import { createMicroApp } from './micro-app';
 import { RmfAppContext } from './rmf-app';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
 
-type DoorCardProps = Omit<BaseDoorCardProps, 'mode'>;
+interface DoorTableRowProps {
+  doorName: string;
+  levelName: string;
+  doorType: number;
+}
 
-const DoorCard = ({ children, ...otherProps }: DoorCardProps) => {
+const DoorTableRow = ({ doorName, levelName, doorType }: DoorTableRowProps) => {
+  const theme = useTheme();
   const rmf = React.useContext(RmfAppContext);
   const [doorState, setDoorState] = React.useState<DoorState | null>(null);
+
   React.useEffect(() => {
     if (!rmf) {
       return;
     }
-    const sub = rmf.getDoorStateObs(otherProps.name).subscribe(setDoorState);
+    const sub = rmf.getDoorStateObs(doorName).subscribe(setDoorState);
     return () => sub.unsubscribe();
-  }, [rmf, otherProps.name]);
+  }, [rmf, doorName]);
+
+  const labelStyle = React.useMemo<SxProps>(() => {
+    const { current_mode } = doorState ?? {};
+    const mode = current_mode?.value;
+
+    const disabled = {
+      color: theme.palette.action.disabledBackground,
+    };
+    const open = {
+      color: theme.palette.success.main,
+    };
+    const closed = {
+      color: theme.palette.error.main,
+    };
+    const moving = {
+      color: theme.palette.warning.main,
+    };
+
+    switch (mode) {
+      case DoorMode.MODE_OPEN:
+        return open;
+      case DoorMode.MODE_CLOSED:
+        return closed;
+      case DoorMode.MODE_MOVING:
+        return moving;
+      default:
+        return disabled;
+    }
+  }, [theme, doorState]);
 
   return (
-    <BaseDoorCard mode={doorState ? doorState.current_mode.value : -1} {...otherProps}>
-      {children}
-    </BaseDoorCard>
+    <TableRow key={doorName}>
+      <TableCell>{doorName}</TableCell>
+      <TableCell>{levelName}</TableCell>
+      <TableCell>{doorTypeToString(doorType)}</TableCell>
+      <TableCell sx={labelStyle}>
+        <Typography
+          data-testid="door-state"
```

Comment:
nit: let's use this opportunity to sanitize double quotes from our code base, only use single-quotes per typescript guidelines

Reply:
I left some comments on google chat about our eslint configuration currently. 
Maybe we can delve into this in a specific PR or it may be the possibility to define conventions. Let me know what you think.

Reply:
sure that sounds like a good idea, it'll be more a maintenance task and should not be prioritized.